scripts/actions/mobskills/tempestuous_upheaval.lua

```diff
@@ -0,0 +1,31 @@
+-----------------------------------
+--  Tempestuous Upheaval
+--
+--  Description: Deals Wind damage to enemies within an area of effect. Additional effect: Knockback
+--  Type: Magical (Wind)
+--  Utsusemi/Blink absorb: 2-3 shadows
+--  Range: 10' radial
+--  Notes: The knockback is rather severe.
+-----------------------------------
+---@type TMobSkill
+local mobskillObject = {}
+
+mobskillObject.onMobSkillCheck = function(target, mob, skill)
+    return 0
+end
+
+mobskillObject.onMobWeaponSkill = function(target, mob, skill)
+    local dmgmod = 1.2
+    -- Assuming you want to calculate damage based on weapon damage and a multiplier
+    local damage = mob:getWeaponDmg() * 3
+
+    damage = xi.mobskills.mobMagicalMove(mob, target, skill, damage * dmgmod, xi.element.WIND, dmgmod, xi.mobskills.magicalTpBonus.MAB_BONUS, 1)
```

Comment:
Why are you multipliying the base damage by the damage modifier and then also passing the damage modifier which will also multiply the damage again?